.gitignore

```diff
@@ -50,7 +50,6 @@ Gemfile.lock
 
 méli-mélo-demos/
 _wetboew-demos/
-~jekyll-dist/
```

Comment:
The goal of that change is to allow me to test out this PR on GitHub Pages since I don't currently have a local Jekyll setup. I plan to remove it once things are ready to merge later on. I snuck in an item for it into https://github.com/wet-boew/GCWeb/pull/1916#issuecomment-1154473960's checklist yesterday.

With the way the GCWeb repo is currently setup, it's super easy to fork it and modify its web pages. But testing out any site template changes is a huge pain IMO. GCWeb doesn't use its include source files directly. It uses "compiled" include files from upstream's ``~jekyll-dist/`` folder. Ditto for layout files and site assets (i.e. CSS/JS/etc).

So to test out my own includes, I need to jump through the following hoops:
1. Modify my fork's "source" include files.
2. Run ``grunt`` to compile my source includes into the ``~jekyll-dist/`` folder
3. Remove this line from ``~jekyll-dist/_includes/settings.liquid``:
```
{%- assign setting-resourcesBasePathTheme = "/dist/GCWeb" -%}{%- assign setting-resourcesBasePathWetboew = "/dist/wet-boew" -%}
```
4. Modify ``.gitignore`` to tell git to track my compiled includes.
5. Modify ``_config.yml`` to tell my GCWeb fork to use its compiled includes instead of upstream's ones.
6. Commit and push to GitHub.
7. Wait around a minute for GitHub Actions to prep things up on GitHub Pages.

Reply:
Gonna mark this as resolved since I replaced that mess of files with these a couple of weeks ago:
* ``_includes/gc-page-feedback/gc-page-feedback.html``
* ``_includes/page-details/footer.html``
* ``sites/gc-page-feedback/includes/css.html``